Cargo.toml

```diff
@@ -29,6 +29,8 @@ serde_json = { version = "0.9", optional = true }
 # Disabled until updated to serde 0.9
 # serde_xml = { version = "0.9.1", optional = true }
 
+clippy = { version = "*", optional = true}
```

Comment:
This needs to be pinned to a specific version, IIRC. Let's call it `">=0.0, <0.1"`